controls/SV-257826.rb

```diff
@@ -15,15 +13,24 @@
 $ sudo dnf remove vsftpd'
   impact 0.7
   ref 'DPMS Target Red Hat Enterprise Linux 9'
-  tag check_id: 'C-61567r925463_chk'
   tag severity: 'high'
+  tag gtitle: 'SRG-OS-000074-GPOS-00042'
   tag gid: 'V-257826'
   tag rid: 'SV-257826r925465_rule'
   tag stig_id: 'RHEL-09-215015'
-  tag gtitle: 'SRG-OS-000074-GPOS-00042'
   tag fix_id: 'F-61491r925464_fix'
-  tag satisfies: ['SRG-OS-000074-GPOS-00042', 'SRG-OS-000095-GPOS-00049', 'SRG-OS-000480-GPOS-00227']
-  tag 'documentable'
-  tag cci: ['CCI-000197', 'CCI-000366', 'CCI-000381']
-  tag nist: ['IA-5 (1) (c)', 'CM-6 b', 'CM-7 a']
+  tag cci: ['CCI-000366', 'CCI-000197', 'CCI-000381']
+  tag nist: ['CM-6 b', 'IA-5 (1) (c)', 'CM-7 a']
+  tag 'host'
+  tag 'container'
+
+  if input('ftp_required')
```

Comment:
why is there an exception for ftp when there is no mention of an isso waiver?  have we had an ask for this in the past?